.travis.yml

```diff
@@ -18,3 +18,26 @@ addons:
     - libkmod-dev
 
 script: autoreconf -fi && ./configure && make distcheck
+
+before_deploy:
+- mkdir dist;
+- export FILES=dist/*
+- mv *.tar.gz dist
+- cd dist
+- sha512sum *.tar.gz > SHA512SUMS
+- sha256sum *.tar.gz > SHA256SUMS
+- sha1sum *.tar.gz > SHA1SUMS
+- md5sum *.tar.gz > MD5SUMS
+- cd -
```

Comment:
shouldn't you export `$FILES` as last thing? because you seem to rely on that there's nothing to expand and `$FILES` gets assigned `dist/*` literally.

i suggest top drop the `$FILES` and put to `deploy` section the actual glob you intend to use:

```
  file: "dist/*"
```

it's more precise and without side effects